cypress/e2e/pages/location.cy.ts

```diff
@@ -142,15 +142,20 @@ describe('Location', () => {
     });
 
     context('when back button is pressed in browser', () => {
```

Comment:
This context says "back button is pressed". Does `cy.visit('/locations/STO-002');` match that description?

Reply:
Yes, the following code shows how it was navigated. The intention of this test case is to  ensure 'awaitingLabware' list on display is updated following an add action on an item  (which is already in awaiting storage) and it is reflected on navigating to another page. So, even if it is not an exact back location, it make sense to any other location, I hope.
      _cy.visit('/locations/STO-002');
      cy.findByText('Rack 1 in Freezer 1 in Room 1234').click();
      cy.findByText('Box 1 in Rack 1 in Freezer 1 in Room 1234').click();_

Reply:
OK as long as you're happy it's testing what it says :+1:

Reply:
I have changed the test case description to _'when navigating to a previous location or another page'_ . Hope that ids makes it more clear